x509-cert/test-support/src/zlint.rs

```diff
@@ -167,17 +171,26 @@ pub fn check_certificate(pem: &[u8], ignored: &[&str]) {
         });
 
     let mut stdout = child.stdout.take().unwrap();
+
+    // read stdout in another thread
```

Comment:
It should not be necessary, moving the `child.wait` down should be enough. There is no need to start a new thread.

Read the output until the end, then wait. The wait was backward, I guess I never noticed because the buffer was enough on my setup, or the output grew and I (and CI) were using an older revision of zlint.